plugins/kubernetes/src/Router.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Entity } from '@backstage/catalog-model';
+import { Route, Routes } from 'react-router-dom';
+
+import { rootCatalogKubernetesRouteRef } from './plugin';
+import { KubernetesContent } from './components/KubernetesContent';
+import { WarningPanel } from '@backstage/core';
+
+const KUBERNETES_ANNOTATION = 'backstage.io/kubernetes';
```

Comment:
so, this annotation name is not very clear as to what it should contain - haven't read the rest of the PR quite yet due to meetings but this should probably be something briefly descriptive - see others in here https://backstage.io/docs/features/software-catalog/well-known-annotations

Reply:
hmmm maybe `backstage.io/kubernetes-ref`? I'm just going to use this to find kubernetes objects that match the current service, wasn't sure what to call it really 🤷‍♂️

Reply:
Oh right, so for now it's actually only used as an on/off switch? Exactly what data is needed to uniquely identify the objects to find in k8s? Would you actually want to use complete label selectors, or is there some convention in place of naming/namespaces/other?

Essentially, I am trying to understand how, given an entity, I am expected to locate k8s objects that match it.

Reply:
Should throw in that I was reviewing this from the point of view that the exact interfaces and integration points are TBD and we'll work them out as we learn more about what models make sense.

Reply:
Yeah good point, I'll chill a bit and let's sketch out the details as we go :)

Reply:
It's being used as an on/off switch for now, but I'm also going to use it as a label selector to find the Kubernetes objects we will show.

However I want to make the label used configurable eventually so that users don't have to go on a kubernetes tagging spree for stuff to show up in backstage.